thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -3044,6 +3039,7 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
             indexPartToSlaveMap.clear();
             dataPartToSlaveMap.clear();
             tlkKeyManagers.kill();
+            contextLogger.reset();
```

Comment:
I think, similar to situation in indexreadslave,
either this can be removed, or there also needs to be a statsUpdater.resetStart().